src/Extensions/InternetExtension.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Xefi\Faker\Extensions;
+
+class InternetExtension extends Extension
+{
+    protected $tld = ['com', 'biz', 'info', 'net', 'org', 'edu', 'gov', 'mil', 'co', 'io'];
+
+    public function sdl(): string
+    {
+        return $this->randomizer->getBytesFromString(
+            implode(range('a', 'z')),
+            $this->randomizer->getInt(5, 10)
+        );
+    }
+
+    public function tld(): string
+    {
+        return $this->pickArrayRandomElement($this->tld);
+    }
+
+    public function domain(): string
+    {
+        return sprintf('%s.%s', $this->sdl(), $this->tld());
+    }
+
+    public function ipv4(): string
+    {
+        return long2ip($this->randomizer->getInt(0, 1) === 0 ? $this->randomizer->getInt(-2147483648, -2) : $this->randomizer->getInt(16777216, 2147483647));
```

Comment:
The negative numbers ensures compatibility with 32 bits systems, but I think we should check all the positive numbers to ensure compatibility with 64 bits systems. Cause `ip2long` (invert of `long2ip`) always return positive numbers so a lot of 64 bits systems are storing IPs as int from `0` to `4294967295`

Reply:
It seems you are right, the documentation is poor on that side, I don't know if I should be avoiding local ips, like in this case, what do you think ?